internal/sdkv2provider/resource_cloudflare_argo.go

```diff
@@ -27,6 +27,11 @@ func resourceCloudflareArgo() *schema.Resource {
 			Cloudflare Argo controls the routing to your origin and tiered
 			caching options to speed up your website browsing experience.
 		`),
+		DeprecationMessage: heredoc.Doc(fmt.Sprintf(`
+			%s field in argo resource is in deprecation phase that will last for one year (Nov 1, 2024).
```

Comment:
do you need a year here? or is the next major release fine?

Reply:
Yes, that's what I'd like to confirm. Next release is good, so what should be the proper wording here?

Reply:
usually something like https://github.com/cloudflare/terraform-provider-cloudflare/blob/03fc7df3bd98ddfc3f351ba1a06ffca1903ab001/internal/sdkv2provider/schema_cloudflare_zone_settings_override.go#L405 is fine. so

`X has been deprecated in favour of ...` is generally suffice. then in the next major release we remove it entirely.